recipes/librepo/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "librepo" %}
+{% set version = "1.11.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rpm-software-management/librepo/archive/{{ version }}.tar.gz
+  sha256: ca30d38ac22a811a27bc86bd28fd2ebc21d3729baa0d7ca15ace248d11507e9b
+
+build:
+  number: 0
+  skip: true  # [win or osx]
```

Comment:
What's the issue with OSX? Feel free to ping me on the feedstock for this.

Reply:
one issue is that `<endian.h>` is not in the same place on OS X (it's in `<platform/endian.h>` or something like this. 

If you want to patch librepo to work on OS X as well be my guest :) otherwise I'll try to take care of this as soon as I have a OS X machine. 

Also, I got librepo to work on Windows using clang-cl! :tada: Once the feedstock is created I'll try to integrate the necessary changes.